### PR TITLE
remove actionbar in PlayerActivity and add exit button to menu

### PR DESCRIPTION
**Pull request by yakagami**
...now in Kotlin ;) (previously #66)

### CODE REVIEW

File: app/src/main/java/rs/ruffle/PlayerActivity.kt [hunk 1]
```diff
@@ -252,4 +260,4 @@ class PlayerActivity : GameActivity() {
             return result
         }
     }
-}
+}
```
Comment (danielhjacobs) [inline app/src/main/java/rs/ruffle/PlayerActivity.kt:263]: The lint says this newline needs to be added back.

File: app/src/main/java/rs/ruffle/PlayerActivity.kt [hunk 2]
```diff
@@ -123,9 +123,16 @@ class PlayerActivity : GameActivity() {
...
                 }
             }
+            val resourceIdExit: Int = items.size + 1
+            menu.add(group, resourceIdExit, Menu.NONE, "Exit")
             popup.setOnMenuItemClickListener { item: MenuItem ->
-                runContextMenuCallback(item.itemId)
-                true
+                if (item.itemId == resourceIdExit) {
+                    finish()
+                    true
```
Comment (torokati44) [inline app/src/main/java/rs/ruffle/PlayerActivity.kt:131]: I guess the `true` can be kept out of the branch, as it's the same in both arms.

File: app/src/main/java/rs/ruffle/PlayerActivity.kt [hunk 3]
```diff
@@ -123,9 +123,16 @@ class PlayerActivity : GameActivity() {
                     item.setChecked(true)
                 }
             }
+            val resourceIdExit: Int = items.size + 1
```
Comment (torokati44) [inline app/src/main/java/rs/ruffle/PlayerActivity.kt:126]: I don't think this has anything to do with resources...? :thinking:
As in, these things:
https://developer.android.com/guide/topics/resources/providing-resources
https://developer.android.com/reference/android/content/res/Resources

**Comment by torokati44** [comment]
I can fix up these minor nits, and rebase/squash your commits before merging, if you don't mind.
It's your `main` branch, but you did leave the "Allow edits by maintainers" box checked after all...

**Comment by yakagami** [comment]
@torokati44 Yes, please make any changes you think are good!

File: app/src/main/java/rs/ruffle/PlayerActivity.kt (hunk 3 above)
Comment (torokati44) [reply app/src/main/java/rs/ruffle/PlayerActivity.kt:126]: Perhaps item?

**Comment by torokati44** [comment]
Turns out, your `main` branch is protected, just like in the main repo, so I wasn't able to force-push to it.
And I accidentally closed this by fumbling around with the `git` CLI, sorry about that! ._.

**Comment by torokati44** [comment]
BTW in the future, it would be better to make a dedicated branch for PRs, so this is avoided. :)

**Comment by torokati44** [comment]
> Turns out, your main branch is protected, just like in the main repo, so I wasn't able to force-push to it.

Okay, just to be precise, I _was_ able to force-push to it, _once_. I intended to push my local `yakagami-main` branch onto your `main` branch. But the command I used for this doesn't work that way. So instead I pushed my own `main` branch onto it, thereby making this PR empty, and closing it. And after this, the branch protection was no longer being overridden by the PR being open, so I was no longer able to force-push to it. :sweat_smile:

**Pull request closed without merging**